components/ThemeSwitcher.js

```diff
@@ -0,0 +1,42 @@
+"use client";
```

Comment:
Copy and Paste, yes?
You don't need this in your project.

Reply:
'use client' just comes into play when you use ServerComponents (especially with next app-router). Since you have a standard project each of your components is by default a 'client component'. Please delete this line.